### DISCUSSION TITLE
Suggest creating a demo to add multiple wallets, but the documentation does not show it. I don't know why

### DISCUSSION THREAD

**Discussion by xixi474** [rainbow-me/rainbowkit#2255] [Q&A] [answered]
Suggest creating a demo to add multiple wallets, but the documentation does not show it. I don't know why

**Comment by DanielSinclair** [comment, marked as answer]
Reference the Custom Wallet List section of the docs here: https://www.rainbowkit.com/docs/custom-wallet-list
